packages/classic/README.md

```diff
@@ -119,11 +117,9 @@ To ensure version predictability, we use [Semantic Versioning](https://semver.or
 
 This package is part of the following suites:
 
-* [Kendo UI for Angular](http://www.telerik.com/kendo-angular-ui/)
-* [KendoReact](http://www.telerik.com/kendo-react-ui/)
-* [Kendo UI for jQuery](http://www.telerik.com/kendo-ui)
-* [UI for ASP.NET MVC](http://www.telerik.com/aspnet-mvc)
-* [UI for ASP.NET Core](http://www.telerik.com/aspnet-core-ui)
+* [Kendo UI for jQuery](https://www.telerik.com/kendo-ui)
+* [UI for ASP.NET MVC](https://www.telerik.com/aspnet-mvc)
+* [UI for ASP.NET Core](https://www.telerik.com/aspnet-core-ui)
 
 All available Kendo UI commercial licenses may be obtained at http://www.telerik.com/purchase/kendo-ui.
```

Comment:
Visit the [Kendo UI Pricing page](https://www.telerik.com/purchase/kendo-ui) to review Kendo UI commercial licensing and pricing.